src/Stratis.Bitcoin/FullNode.cs

```diff
@@ -217,12 +217,11 @@ public void Stop()
 
             this.ConnectionManager.Dispose();
 
-            foreach (IDisposable dispo in this.Resources)
-                dispo.Dispose();
+            foreach (IDisposable disposable in this.Resources)
```

Comment:
please dispose of this at the end 
`  this.ConnectionManager.Dispose();`
and right after it dispose of 
`this.nodeSettings.LoggerFactory.Dispose()`